### PR TITLE
Adds ipv4 address to wslfetch

### PR DESCRIPTION
**Pull request by ThatWeirdAndrew**
## Description

Added ipv4 address to wslfetch info as suggested in #139 
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have read Code of Conduct and Contributing documentations.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: src/wslsys.sh [hunk 1]
```diff
@@ -116,6 +116,10 @@ function get_wsl_packages() {
 	echo "$packages"
 }
 
+function get_wsl_ip() {
+	echo "$(hostname -I)"
```
Comment (patrick330602) [inline src/wslsys.sh:120]: busybox `hostname` command does not have -I parameters. What are the additional required packages for Alpine here?

**Review by patrick330602 — CHANGES_REQUESTED** [review]

File: src/wslsys.sh (hunk 1 above)
Comment (SuperSandro2000) [reply src/wslsys.sh:120]: ``net-tools` neither so this code does not work for alpine.

Comment (ThatWeirdAndrew) [reply src/wslsys.sh:120]: `ip -4 -o addr show eth0 | awk '{print $4}' | cut -d "/" -f 1` seems to work on AlpineWSL. Will it work?

Comment (ThatWeirdAndrew) [reply src/wslsys.sh:120]: It also works on Debian, Ubuntu, Gentoo, and Kali.

Comment (patrick330602) [reply src/wslsys.sh:120]: It requires `iproute`/`iproute2` packages for some distributions (WLE requires `iproute` and Arch requires `iproute2` in my testing). You should add them in the build scripts, thank you.

Comment (ThatWeirdAndrew) [reply src/wslsys.sh]: I have tested the command on all the distributions listed in the `configure.sh` and have updated it accordingly.

**Review by patrick330602 — APPROVED** [review]

**Pull request merged**